MementarTimestamp stamp
string id
string subject
string predicat
string object
bool added